setup.cfg

```diff
@@ -74,6 +79,7 @@ server =
     suds-jurko
     tornado ~=5.1.1
     tornado-m2crypto
+    importlib_resources
```

Comment:
should this be also in the conda env files?

Reply:
Possibly, though this is a backport of the Python 3.9 standard library so I might bump the Python version instead given there is little reason to care about Python 3.8.

Reply:
Yes, I think it's early enough that you can bump the version in DIRACOS2 in the next patch release (I would not do it so easily if it was production code, but as of now it's still experimental)

Reply:
I agree. On that note I've made an issue with some thoughts about handling this when DIRACOS2 is in production: https://github.com/DIRACGrid/DIRACOS2/issues/30